### PR TITLE
Add calculator components

### PR DESCRIPTION
**Pull request by adamilare**
In this update, the following were implemented;
- Make calculator component in new directory
- Add calculator's sub-components
- Style calculator's components

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 import reportWebVitals from './reportWebVitals';
+import Calculator from './components/calculator';
+import './css/calculator.css';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
   </React.StrictMode>,
```
Comment (franclobo) [inline src/index.js]: - [ ] The Calculator component should be imported into the App.js (main) component after that the App component will be imported into the index.js file. Please 🙏🏽 refactor your code to delete all the boilerplate that got created by the CRA (texts, images, and styles) in the App.js file for organized code.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @adamilare, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️ 
- Correct git flow ✔️
- Good UI ✔️


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (adamilare) [reply src/index.js]: Thank you for pointing that out

Comment (adamilare) [reply src/index.js]: @franclobo Can you please complete the review on the dashboard? It is still showing "review pending".
Thanks

**Review by gemmen29 — APPROVED** [review]
Hi @adamilare, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**